layout/_macro/post.swig

```diff
@@ -78,11 +78,16 @@
             #}</time>
             {% endif %}
 
+            {% set date_updated_diff = date(post.date, config_date_format) != date(post.updated, config.date_format) %}
             {% if theme.post_meta.created_at and theme.post_meta.updated_at %}
-              <span class="post-meta-divider">|</span>
+              {% if !theme.post_meta.updated_diff || theme.post_meta.updated_diff &&  date_updated_diff %}
+                {% set display_updated = true %}
+              {% endif %}
             {% endif %}
 
-            {% if theme.post_meta.updated_at %}
+            {% if display_updated %}
```

Comment:
It will cause problems when the user choose only to show the `updated_at` and not the `created_at`.

Reply:
You're right, thanks. Just revised and verified expected behavior with `created_at`, `updated_at`, and `updated_diff`, by themselves and together.